ped/__init__.py

```diff
@@ -147,6 +148,13 @@ def find_file(obj: Any) -> Optional[str]:
                 pass
     except Exception:
         pass
+
+    if os.environ.get("PED_OPEN_DIRECTORIES"):
+        fname_path = Path(str(fname))
```

Comment:
```suggestion
        fname_path = Path(fname)
```

Reply:
Fails on the mypy static check if you remove the str cast.

Reply:
Ah, probably because `fname` can be `None` at this point. So it'd probably be more correct to do

```python
if fname and os.environ.get(...):
    fpath = Path(fname)
```